radionets/evaluation/utils.py

```diff
@@ -6,6 +6,7 @@
 import torch.nn.functional as F
 from torch.utils.data import DataLoader
 import h5py
+from numba import float64, vectorize, set_num_threads
```

Comment:
Is `float64` actually used here? My flake8 says otherwise

Reply:
It is actually not and I though I removed it :sweat_smile: 

It would be needed if we would decorate a function for jit. With the vectorize decorator the call-signature-sting gets interpreted.